src/test/ui/proc-macro/derive-bad.stderr

```diff
@@ -5,6 +5,10 @@ LL | #[derive(A)]
    |          ^ expected `:`
    |
    = note: this error originates in the derive macro `A` (in Nightly builds, run with -Z macro-backtrace for more info)
+help: consider using `enum` instead of `struct`
+   |
+LL | #[derive(enum)]
+   |          ~~~~
```

Comment:
Seems like this shouldn't happen, the suggested replacement has 0 chance to work at all.

Reply:
I fixed it.